Yosemite/YosemiteTests/Tools/Media/MediaImageExporterTests.swift

```diff
@@ -0,0 +1,84 @@
+import MobileCoreServices
+import XCTest
+@testable import Yosemite
+
+final class MediaImageExporterTests: XCTestCase {
+    func testExportingAnImageWithTypeHint() {
+        let mockData = UIImage(named: "image", in: Bundle(for: type(of: self)), compatibleWith: nil)!.pngData()
```

Comment:
If this is supposed to use the `nara_320.jpg`, I found that the tests still pass even if the image is not there. Am I missing something? 🤔

Reply:
I renamed the image name in the asset catalog to "image", as indicated in the file path `Yosemite/YosemiteTests/Media.xcassets/image.imageset/Contents.json` - it's confusing though, I'm going to rename the image file to `image`

Reply:
So what I did was I deleted the image in `Media.xcassets` and then ran the tests. The tests still passed. Maybe it's just me? 🤔

Reply:
have you run the test with the image in the asset catalog before deleting the image? if you delete the derived data, do you still see the test passing? I was able to reproduce this behavior and after some experiments, I suspect that images in the asset catalog in a non-main bundle stay in the derived data even after deleting them in Xcode. see if this is the case for you!


also, I realized I didn't answer your first question clearly:

> If this is supposed to use the nara_320.jpg, I found that the tests still pass even if the image is not there. Am I missing something? 🤔

the image filename doesn't have to match the name in the asset catalog and thus the name we call in `UIImage(named:*)`. What I did in the first place was to import an image file "nara_320.jpg" to the asset catalog, and then rename the image asset to "image" in the asset catalog. Later in https://github.com/woocommerce/woocommerce-ios/pull/1776/commits/1c09d045178a08b241f30def32aeba13f154b4f5, I deleted the "nara_320.jpg" image from the asset catalog under "image" and then import an image file "image.jpg" 🙂

Reply:
Ah, you're right. It is the derived data. Sorry, I should have suspected that. 🤦‍♂